src/validator/test/QueryValidatorTest.cpp

```diff
@@ -1237,5 +1238,74 @@ TEST_F(QueryValidatorTest, TestMaxAllowedStatements) {
               "SemanticError: The maximum number of statements allowed has been exceeded");
 }
 
+TEST_F(QueryValidatorTest, FindPath) {
+    // TODO: Implement the plan.
+    // shortest
+    {
+        std::string query = "FIND SHORTEST PATH FROM \"1\" TO \"2\" OVER like UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query = "FIND SHORTEST PATH FROM \"1\" TO \"2\",\"3\" OVER like UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query =
+            "FIND SHORTEST PATH FROM \"1\",\"2\" TO \"3\",\"4\" OVER like UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {};
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query = "FIND SHORTEST PATH FROM \"1\" TO \"2\" OVER like, serve UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query =
+            "YIELD \"1\" AS src, \"2\" AS dst"
+            " | FIND SHORTEST PATH FROM \"1\" TO \"2\" OVER like, serve UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+
+    // all
+    {
+        std::string query = "FIND ALL PATH FROM \"1\" TO \"2\" OVER like UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query = "FIND ALL PATH FROM \"1\" TO \"2\",\"3\" OVER like UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query = "FIND ALL PATH FROM \"1\",\"2\" TO \"3\",\"4\" OVER like UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query = "FIND ALL PATH FROM \"1\" TO \"2\" OVER like, serve UPTO 5 STEPS";
+        std::vector<PlanNode::Kind> expected = {
+        };
+        EXPECT_TRUE(checkResult(query, expected));
+    }
+    {
+        std::string query = "YIELD \"1\" AS src, \"2\" AS dst"
+                            " | FIND ALL PATH FROM \"1\" TO \"2\" OVER like, serve UPTO 5 STEPS";
```

Comment:
`FIND ALL PATH FROM $-.src TO $-.dst OVER like, serve UPTO 5 STEPS` ?

Reply:
Sorry, it was a mistake.